e2e/specs/httpCache.cy.js

```diff
@@ -0,0 +1,262 @@
+describe('HTTP cache tests', () => {
```

Comment:
The iframe support in cypress is still very buggy and was quite hassle to get it to work. And the moment it seems to be ok. But if notice this to be too flaky, we might need an alternative solution to check the emails.

Reply:
We are implicitly testing maildev's UI here... This would be better to do with maildev's [REST API](https://github.com/maildev/maildev/blob/master/docs/rest.md) (accessible under localhost:3000/mail/email)

A REST API was a criterion we were explicitly looking for when switching to maildev in #3528, exactly for purposes like this.

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/3610/commits/ae74c7bf46bd9f8bb53d96d6ca6d5f42790ce79c